static/api-extractor.json

```diff
@@ -0,0 +1,340 @@
+/**
```

Comment:
Should this file be commited? seems like it'll be auto-generated each build right?

Reply:
Yes, this should be commited.
That file contains configuration to be used by api-extractor to bundle the types into a single file